test/Typewriter.Test/MultipleNamespacesTestRunner.cs

```diff
@@ -54,7 +54,7 @@ string getMetadataFile(TestLanguage testLanguage)
 
             // Arrange
             var languageStr = language.ToString();
-            var directoryPostfix = isPhpBeta ? "PHPBeta" : languageStr;
+            var directoryPostfix = languageStr + (isBeta ? "Beta" : string.Empty);
```

Comment:
not sure if case sensitivity matters here but for php would languageStr be "PHP" so the string ends up exactly the same. Since it's a passed in parameter i don't know that's guaranteed and if it matters.

Reply:
i think this is answered looking at how it's called from the PHPMultipleNamespaces file.

Reply:
This is fed from an enum that is local to the tests, so nobody will be able to change it by changing a string parameter somewhere. The test caller needs to specify an enum value.

Also this is only used as part of a directory name where case doesn't matter as long as you are on Windows :)